Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -736,7 +1059,19 @@ public class RealmServer: NSObject {
                 "title": "SwiftPerson"
             ],
                 "relationships": [:]
-        ], failOnError)
+        ]
+
+        let rules = app.services[serviceId].rules
+
+        rules.post(on: group, dogRule, failOnError)
+        rules.post(on: group, personRule, failOnError)
+        rules.post(on: group, hugeSyncObjectRule, failOnError)
+        // When running ObjcObjectServerTests,
+        // we do not want to pull in swift schema reqs
+        #if SWIFT_PACKAGE && REALM_HAVE_COMBINE
```

Comment:
Wont using these macros mean this class will cause issues when running tests from the xcproj?